website/src/utils/getAvatar.ts

```diff
@@ -0,0 +1,10 @@
+// Util function for fetcing the author's avatar image
+const getAvatar = (author: string) => {
+    const authorAvatar = `../Images/blog/authors/${author
+        .toLowerCase()
```

Comment:
it would be great if we extract only this

```
author.toLowerCase().replace(/[^\w ]+/g, "").replace(/ +/g, "-")
``` 
and passing `author` as props

this way, code will be modular

Reply:
have done the change, and also as the Image path was used at many places so, added it in the util file for better readability.